README.md

```diff
@@ -2,6 +2,23 @@
 
 This python project is a helper package that uses power analysis to calculate required sample size for any experiment.
 
+# Get Started
+```buildoutcfg
+pip install sample-size
+run-sample-size
+```
+
+# Notes
```

Comment:
Wonder if this can be more specific than just Notes. Also we should move this to one level down in markdown headers similar to the above comment.

Reply:
I'll update it to Script Constraints.